src/Kunstmaan/MediaBundle/Repository/FolderRepository.php

```diff
@@ -107,7 +107,8 @@ public function getAllFolders($limit = null)
     {
         $qb = $this->createQueryBuilder('folder')
             ->select('folder')
-            ->where('folder.parent is null AND folder.deleted != true')
+            ->where('folder.parent is null AND folder.deleted != :deleted')
+            ->setParameter('deleted', true, \PDO::PARAM_BOOL)
```

Comment:
Same question as in `src/Kunstmaan/AdminBundle/Repository/ExceptionRepository.php`. If it's not required I would remove the 3th parameter as we don't set it anywhere else.

Reply:
same answer: PARAM_BOOL's have been removed

Reply:
Yep, has been removed